src/main/java/edu/cornell/kfs/sys/CUKFSPropertyConstants.java

```diff
@@ -44,5 +44,8 @@
     public static final String DOCUMENT_FAVORITE_ACCOUNT_LINE_IDENTIFIER = "document.favoriteAccountLineIdentifier";
 
     public static final String AWARD_EXTENSION_BUDGET_ENDING_DATE = "extension.budgetEndingDate";
+    
+    public static final String RECURRING_DV_PARTIAL_TRANSACTION_COUNT_FIELD_STARTER = "document.sourceAccountingLine[";
```

Comment:
Personally, I don't find this name very descriptive, and I also wonder if there isn't already a constant for this, or at least the parts - "document" and "sourceAccountingLine" that could be used and concatenated together.